recipe/meta.yaml

```diff
@@ -11,27 +11,36 @@ source:
 
 build:
   noarch: python
+  skip: true # [py<37]
```

Comment:
If I'm not mistaken, Anaconda only provide support for Python 3.8 and newer.

Reply:
The team prefers to try and match upstreams requirements regardless of our supported python versions.

Reply:
Multiple people have told me to start putting this in recipes. I'm guessing that's the reasoning, but nobody has given me a reason as to why we're doing this.

It smells like it needs to be a linter rule to me

Reply:
The recipe should reflect how the package can be built, not what we currently support. If support were an indicator, we would have to write `skip: true #  [py<38 and py>311]` into every single recipe that builds a python package. And update it with every python version we deprecate and/or add.

The python version we build with are selected via the `conda_build_config.yaml`. That way, we separate the way we can build the package according to upstream (`meta.yaml`) and how we actually build the package for Anaconda (`conda_build_config.yaml`).